test/helpers/kubectl.go

```diff
@@ -67,7 +68,7 @@ func (kub *Kubectl) ExecPodCmd(namespace string, pod string, cmd string) (string
 	command := fmt.Sprintf("%s exec -n %s %s -- %s", kubectl, namespace, pod, cmd)
 	stdout := new(bytes.Buffer)
 	stderr := new(bytes.Buffer)
-	exit := kub.Execute(command, stdout, stderr)
+	exit, _ := kub.Execute(command, stdout, stderr)
```

Comment:
Why are we ignoring the error?

Reply:
In this case, we only need if the exit was success or not. The error message is not essential.(A lot of them are supposed to fail)